python311Packages.hahomematic: 2023.7.3 -> 2023.8.2

fabaff opened the pull request:
## Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

fabaff committed:
python311Packages.hahomematic: 2023.7.0 -> 2023.7.3

Changelog: https://github.com/danielperna84/hahomematic/releases/tag/2023.7.3

fabaff committed:
python311Packages.hahomematic: 2023.7.3 -> 2023.8.2

Diff: https://github.com/danielperna84/hahomematic/compare/refs/tags/2023.7.3...2023.8.2

Changelog: https://github.com/danielperna84/hahomematic/releases/tag/2023.8.2

fabaff commented:
Result of `nixpkgs-review pr 247885` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>python310Packages.hahomematic</li>
    <li>python310Packages.hahomematic.dist</li>
  </ul>
</details>
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python311Packages.hahomematic</li>
    <li>python311Packages.hahomematic.dist</li>
  </ul>
</details>

tjni commented:
1. The latest version requires Python 3.11; can the `disabled` property be added?
2. Would it be possible to update to the latest version? It will fix the build on staging-next, since it updates setuptools and wheel in pyproject.toml.
3. As a favor, can you also add wheel to nativeBuildInputs?

fabaff committed:
python311Packages.hahomematic: 2023.8.2 -> 2023.8.3

Diff: https://github.com/danielperna84/hahomematic/compare/refs/tags/2023.8.2...2023.8.3

Changelog: https://github.com/danielperna84/hahomematic/releases/tag/2023.8.3

fabaff committed:
python311Packages.hahomematic: enable tests

fabaff committed:
python311Packages.hahomematic: 2023.8.3 -> 2023.8.4

Diff: https://github.com/danielperna84/hahomematic/compare/refs/tags/2023.8.3...2023.8.4

Changelog: https://github.com/danielperna84/hahomematic/releases/tag/2023.8.4

fabaff committed:
python311Packages.hahomematic: 2023.8.4 -> 2023.8.5

Diff: https://github.com/danielperna84/hahomematic/compare/refs/tags/2023.8.4...2023.8.5

Changelog: https://github.com/danielperna84/hahomematic/releases/tag/2023.8.5

fabaff committed:
python311Packages.hahomematic: 2023.8.5 -> 2023.8.6

Diff: https://github.com/danielperna84/hahomematic/compare/refs/tags/2023.8.5...2023.8.6

Changelog: https://github.com/danielperna84/hahomematic/releases/tag/2023.8.6

fabaff committed:
python311Packages.hahomematic: 2023.8.6 -> 2023.8.7

Diff: https://github.com/danielperna84/hahomematic/compare/refs/tags/2023.8.6...2023.8.7

Changelog: https://github.com/danielperna84/hahomematic/releases/tag/2023.8.7

fabaff committed:
python311Packages.hahomematic: 2023.8.7 -> 2023.8.9

Diff: https://github.com/danielperna84/hahomematic/compare/refs/tags/2023.8.7...2023.8.9

Changelog: https://github.com/danielperna84/hahomematic/releases/tag/2023.8.9

fabaff commented:
Result of `nixpkgs-review pr 247885` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python311Packages.hahomematic</li>
    <li>python311Packages.hahomematic.dist</li>
  </ul>
</details>
